fog/ingest/client/src/config.rs

```diff
@@ -72,4 +72,21 @@ pub enum IngestConfigCommand {
     /// Retrieves a private key from a remote ingest enclave and sets it as
     /// the current enclaves's private key.
     SyncKeysFromRemote { peer_uri: String },
+
+    ///  Retrieves the ingress public keys for the entire system (as opposed to
+    ///  those of a single IngestServer) and filters according to the provided
+    ///  parameters.
+    GetIngressPublicKeyRecords {
+        /// Ingress keys are "started" at certain blocks. Only ingress keys that
+        /// are "started"  at this block index will be included in the response.
+        #[structopt(short = "s", long = "start-block-at-least")]
```

Comment:
If this is staying then is it possible to make it default to 0?

Reply:
Done.